src/sdk.rs

```diff
@@ -4,6 +4,7 @@ use borsh::{BorshDeserialize, BorshSerialize};
 
 const READ_STORAGE_REGISTER_ID: u64 = 0;
 const INPUT_REGISTER_ID: u64 = 0;
+const GAS_FOR_STATE_MIGRATION: u64 = 100_000_000_000_000;
```

Comment:
I'm not 100% sure on the value for this constant. 100 Tgas seems like it should be enough any sort of state migration we want to do, but we could push it higher if we think this might not be enough.

We could also leave it as is and update the constant only if we actually write a migration that requires more gas (I assume we'll test all upgrades before we do them, so we'll know if we need more gas).

Reply:
I think it makes sense to set it as a max value (300 Tgas). As this is not user's function I don't think there's any problem that we will be required to pay some more gas for the state migration (which hopefully won't happen too frequently). So I don't think we need to put any max value constrain here.

Reply:
Well, we can't set it to 300 Tgas because that the way promises work is they take gas from the original transaction. So if the original `deploy_upgrade` transaction was given 300 Tgas then we could only give the promise it creates an amount of gas equal to 300 minus whatever amount is used in creating the promise.

We could set it higher, e.g. I think 250 Tgas would work, but I don't know if there is much value in figuring out exactly how high it can be when we don't know that 100 isn't enough.

Reply:
Oh, yeah sorry I forgot that we are talking about the inner promise. Then sure 250 Tgas could be an option.

Reply:
Done in https://github.com/aurora-is-near/aurora-engine/pull/103/commits/208de2d0297e0658c680feea697bf5ff953bc25b